### ISSUE TITLE
add copyright header to a few sandbox/ scripts

### ISSUE THREAD

**Issue by ctb** [ged-lab/khmer#797]
See:
```
normalize-by-align.py
read_aligner.py
slice-reads-by-coverage.py
```

**Comment by ctb** [comment]
All of these can use '2014'; see other scripts in sandbox/ for the general format.

**Comment by alameldin** [comment]
I am working on this issue

**Comment by alameldin** [comment]
ready for review

**Comment by ctb** [comment]
Hi @alameldin, we need the checklist and a ChangeLog entry, please!

**Comment by alameldin** [comment]
Hi @ctb , i pulled a new request including the copyright, checklist and ChangeLog entry.
it took me some time to figure out the git troubleshooting.
Thanks!

**Comment by alameldin** [comment]
Ready for review

**Comment by mr-c** [comment]
Fixed by #826
